fcrepo-clients/src/main/java/edu/unc/lib/dl/fcrepo4/ContentObject.java

```diff
@@ -28,18 +35,35 @@
 
 	protected ContentObject(PID pid, Repository repository, RepositoryObjectDataLoader dataLoader) {
 		super(pid, repository, dataLoader);
-		// TODO Auto-generated constructor stub
 	}
 
 	public void addChild(ContentObject child) {
 		// TODO
 	}
-	
+
 	public BinaryObject addDescription(InputStream contentStream) {
 		return null;
 	}
-	
+
 	public BinaryObject getDescription() {
 		return null;
 	}
+
+	/**
+	 * Retrieve a list of member content objects for this object.
+	 * 
+	 * @return
+	 */
+	public List<ContentObject> getMembers() {
+		List<ContentObject> members = new ArrayList<>();
+		Resource resc = getResource();
+
+		for (StmtIterator it = resc.listProperties(PcdmModels.hasMember); it.hasNext(); ) {
```

Comment:
Oh, didn't realize that.  I don't have a strong opinion on it, I switched to for-loops after a previous PR you commented on.  Most examples I saw at the time were using for-loops, but the only real reason I've seen to prefer one over the other is the first answer here:
http://stackoverflow.com/questions/99164/iterate-with-for-loop-or-while-loop